docs/source/reference/ai/custom-ai-function.rst

```diff
@@ -39,47 +41,102 @@ An abstract method that must be implemented in your child class. The setup funct
   - True: Batching should be enabled
   - False: Batching is disabled.
 
-The custom setup operations for the function can be written inside the function in the child class. If there is no need for any custom logic, then you can just simply write "pass" in the function definition.
-
-Example of a Setup Function
+Any additional arguments needed for creating the function must be passed as arguments to the setup function. *(Please refer to the ChatGPT function example)*
```

Comment:
Can we give a link to the `ChatGPT function example`?

Reply:
done